tests/output/printers/LogPrinterTest.py

```diff
@@ -32,7 +32,7 @@ def test_logging(self):
         uut.log_message(self.log_message, end="")
         self.assertEqual(uut.printer.string, str(self.log_message))
 
-        uut = LogPrinter(StringPrinter(), log_level=LOG_LEVEL.DEBUG)
+        uut = LogPrinter(StringPrinter(), log_level=LOG_LEVEL.INFO)
```

Comment:
This is the cause of the failing test! :D
The `log_level` parameter sets the minimum log-level to print. `DEBUG` is the lowest one, `INFO` follows. So please don't change this line, then tests work fine :D